src/AsyncCreatable.js

```diff
@@ -7,19 +7,42 @@ const AsyncCreatable = React.createClass({
 	render () {
 		return (
 			<Select.Async {...this.props}>
-				{(asyncProps) => (
-					<Select.Creatable {...this.props}>
-						{(creatableProps) => (
-							<Select
-								{...asyncProps}
-								{...creatableProps}
-							/>
-						)}
+				{({ref, ...asyncProps}) => {
+					const asyncRef = ref
+					return <Select.Creatable {...asyncProps} >
+						{({ref, ...creatableProps}) => {
+							const creatableRef = ref
+							return this.props.children({
+								...creatableProps,
+								ref: (select) => {
+									creatableRef(select)
+									asyncRef(select)
+									this.select = select
+								}
+							})
+						}}
```

Comment:
There are a bunch of cases where it's handy to be able to inject a child - e.g. testing, adding custom behavior that isn't natively present. I've taken the pattern from the implementation in `Async`.